code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -33,6 +33,7 @@ This saves us from having to call add_fingerprint() any time something is put in
 	if(!W)
 		return FALSE
 	if(put_in_active_hand(W) || put_in_inactive_hand(W))
+		world << "Successfully put in hand"
```

Comment:
clean up your debug messaaaages

Reply:
woops!
Mark it "Changes requested" to remind me to fix this